install/install.js

```diff
@@ -1,3 +1,6 @@
 const install = require('../src/install');
+const exec = require('@actions/exec')
 
+
+exec.exec('echo "TELEPRESENCE_GITHUB_ACTION_INTEGRATION=true" >> $GITHUB_ENV')
```

Comment:
shouldn't the env var name start with `TELEPRESENCE_REPORT_` ? so it's included in the meta of the env TP is reporting?